frontend/src/pages/profile/libs/enums/activity-threshold.enum.ts

```diff
@@ -0,0 +1,7 @@
+const ActivityThreshold = {
+  FIRST: 0,
+  SECOND: 1,
+  THIRD: 3,
+} as const;
```

Comment:
Shouldn't THIRD has value of 2?

Reply:
No, because these are user activity breakpoints, but I checked that should be 5, not 3)